### PR TITLE
add: subscription section

### PR DESCRIPTION
**Pull request by GustavoUT22**
Check it out!

### CODE REVIEW

File: src/pages/Blog/Blog.tsx [hunk 1]
```diff
@@ -14,17 +20,37 @@ const Blog: React.FC = () => {
...
       </div>
       <div>Pagination component</div>
-      <div>
-        <div>
-          <h2>Suscríbete</h2>
-          <p>deseas estar mas pendiente de más articulos</p>
-          <div>
-            <input type="email" />
-            <button>Suscríbete</button>
+      <Section type="margin" size="sm">
```
Comment (CrhistianT7) [inline src/pages/Blog/Blog.tsx:23]: move all this section to another component, to keeo clean blog main page

Comment (GustavoUT22) [reply src/pages/Blog/Blog.tsx]: done

File: src/pages/Blog/Blog.tsx [hunk 2]
```diff
@@ -14,17 +17,7 @@ const Blog: React.FC = () => {
...
-          <h2>Suscríbete</h2>
-          <p>deseas estar mas pendiente de más articulos</p>
-          <div>
-            <input type="email" />
-            <button>Suscríbete</button>
-          </div>
-        </div>
-        <div>img suscríbete</div>
-      </div>
+      <Subscribe text={text} setText={setText} />
```
Comment (CrhistianT7) [inline src/pages/Blog/Blog.tsx:20]: move state management  inside it sown component

Comment (GustavoUT22) [reply src/pages/Blog/Blog.tsx]: done

**Review by CrhistianT7 — APPROVED** [review]
Looks Good To Me!

**Pull request merged**